src/newexample.cpp

```diff
@@ -0,0 +1,13 @@
+#include <cstdio>
+
+int main(int argc, char **argv)
+{
+	if(argc > 3) 
+		fprintf(stderr, "usage: clang-tidy-example [1/2]\n");
+
+	fprintf(stderr, "usage: clang-tidy-example [1/2]\n");
+	
+	
```

Comment:
![Codacy](https://app.staging.codacy.org/assets/images/favicon.png) Issue found: [use a trailing return type for this function](https://app.staging.codacy.org/manual/josemiguelmelo/sample-clangtidy/pullRequest?prid=9421)